src/main/java/ToDos.java

```diff
@@ -0,0 +1,6 @@
+public class ToDos extends Task{
```

Comment:
Class names should be singular, Deadlines.java and Events.java have the same issue

Reply:
thanks